pexpect/spawnbase.py

```diff
@@ -140,6 +147,15 @@ def _coerce_send_string(self, s):
             return s.encode('utf-8')
         return s
 
+    def get_buffer(self):
```

Comment:
Let's make these private methods (`_` prefix), so people don't start using them directly and cause us another backwards compatibility headache.